scripts/build-bundle

```diff
@@ -17,20 +23,13 @@ source ${SCRIPTS_DIR}/lib/image
 source ${SCRIPTS_DIR}/hack/patch-rancher-monitoring
 source ${SCRIPTS_DIR}/hack/patch-rancher-logging
 source ${SCRIPTS_DIR}/hack/patch-rancher-monitoring-crd
-source ${SCRIPTS_DIR}/hack/patch-harvester-chart
```

Comment:
Should the above line be removed here? It looks arm-related to me.

Reply:
good spot. i think i deleted it. fixed it up now.